hs/src/Reach/AST/DL.hs

```diff
@@ -183,6 +183,15 @@ data DLBlock
 instance Pretty DLBlock where
   pretty (DLBlock _at _ ss da) = prettyBlock (render_dls ss) da
 
+data DLValue
```

Comment:
I think this and `DLArgExpr` are only used by `Core`, so maybe it should be in `Eval/Types`?